src/testing/loadtest-azure/scripts/README.md

```diff
@@ -0,0 +1,13 @@
+# Azure Load Test - Scripts
+
+## Overview
+
+Script | Description
+-- | --
+create-loadtest.ps1 | Create new load tests in an existing Azure Load Test service instance
```

Comment:
IMHO listing files like this doesn't add any value and only makes it more difficult to keep up to date. How about actually describing how are they expected to be used/triggered, what prerequisites need to be met etc.?

Reply:
Some of these files are there since the beginning - we'll clean up the `scripts/` directory before we merge this. All the `*-harness.ps1` files can also be removed.